src/browser/setupWorker/start/createResponseListener.ts

```diff
@@ -3,7 +3,7 @@ import {
   SetupWorkerInternalContext,
 } from '../glossary'
 import { ServiceWorkerMessage } from './utils/createMessageChannel'
-
+import { isResponseWithoutBody } from '@mswjs/interceptors'
```

Comment:
Nitpick: a new line after the import would be nice. Are by any chance not running Prettier?

Reply:
I usually have it running, maybe it just lagged in vscode?